### PR TITLE
Relnotes for Desktop 2.1.0.1 release

### PR DESCRIPTION
**Pull request by usha-mandya**
Added release notes for Desktop 2.1.0.1 release.

- Replaced 2.1.0.0 section with 2.1.0.1
- Added new bug fixes
- Removed the known issue (docker/for-win#4343) from the Windows release notes as this issue is now fixed
- Added a note that users must sign in and create a docker ID to download Docker Desktop.

### CODE REVIEW

**Comment by usha-mandya** [comment]
@mat007 @nebuk89 - Could you take a look at the updated release notes:

- https://deploy-preview-9215--docsdocker.netlify.com/docker-for-windows/release-notes/
- https://deploy-preview-9215--docsdocker.netlify.com/docker-for-mac/release-notes/

File: docker-for-mac/release-notes.md [hunk 1]
```diff
@@ -58,6 +60,9 @@ Docker Desktop Community 2.1.0.0 contains the following experimental features.
 
 ### Bug fixes and minor changes
 
+* Docker Desktop now allows users to expose privileged UDP ports. [docker/for-mac#3775](https://github.com/docker/for-mac/issues/3775)
+* Fixed an issue where running some Docker commands can fail if you are not using Credential Helpers. [docker/for-mac#3785](https://github.com/docker/for-mac/issues/3785)
+* Updated Hyperkit to version [0.20190802](https://github.com/moby/hyperkit/releases/tag/v0.20190802).
```
Comment (mat007) [inline docker-for-mac/release-notes.md]: Should this one be in the upgrades section?

Comment (usha-mandya) [reply docker-for-mac/release-notes.md]: Yes, that's right. I'll move this to the upgrades section.

**Comment by usha-mandya** [comment]
> Should we not keep the 2.0.0.0 section?

That's a good question. I thought we are combining both sections. Let me double-check with Ben.

**Comment by usha-mandya** [comment]
@DawnWood-Docker - Could you take a look at the updated relnotes? This is scheduled to go GA today.

**Pull request merged**